src/addMeToYourFriends.ts

```diff
@@ -0,0 +1,146 @@
+import { html, TemplateResult } from "lit-html";
+import { styleMap } from "lit-html/directives/style-map";
+import { DataBrowserContext, LiveStore } from "pane-registry";
+import * as UI from "solid-ui";
+import { padding, textCenter } from "./baseStyles";
+
+let buttonContainer = <HTMLDivElement>document.createElement("div");
+//TODO create dedicated component in UI
+const positiveFrontendMessageDiv = <HTMLDivElement>document.createElement("div");
+positiveFrontendMessageDiv.setAttribute(
+  "style",

```

Comment:
In this case maybe it's appropriate to add a message style to baseStyles and add it in the same way as styles = .. styleMap.  If there are specific styles to friends you can also create another file called perhaps friendStyles.

Reply:
Here, see @timbl comment in Gitter regarding the styles should be from solid-ui and use setStyle().  I will see if I can find an example and add here.  The file to find them in is solid-ui/src/style.js and you can add a style here that you need. I see something called messageBodyStyle...  this is something to look at although I don't know if it's exactly what you need.  I will do some more research for this as well, as we definitely have errors in other places. I'm going to make another ticket though for styles in general as to make it easier to read in solid-ui I think we should perhaps have more than one style.js file going forward.

Reply:
I actually couldn't find many examples but here is one    ` head = table.appendChild(dom.createElement('tr'));
            style.setStyle(head, 'autocompleteRowStyle'); // textInputStyle or`.  I also think solid-ui/src/messageArea.js is worth checking out for error messages.  I can't be sure about this but something to look into.

Reply:
regarding styles, i did not change so much because it will cause inconsistencies on the Profile Pane. I propose opening an additional ticket with topic - align profile pane styles with UI.